conda/recipes/rapids_core_dependencies/meta.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 39) %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.5').split('.')[:2]) %}
+{% set cuda_major=cuda_version.split('.')[0] %}
+
+package:
+  name: rapids_core_dependencies
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
```

Comment:
Did you copy this from somewhere? Typically build numbers are incremented manually, in my experience. Also `{{ this_syntax }}` inserts a jinja template variable (like those set by `{% set ... %}` above), while [`GIT_DESCRIBE_NUMBER` is an environment variable](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#git-environment-variables).

Reply:
I lifted this from `librmm` ( https://github.com/rapidsai/rmm/blob/branch-22.12/conda/recipes/librmm/meta.yaml#L44 ) as this is the first conda package I have written.

Reply:
Alright. Let's leave it as-is, because it seems to work (a bit contrary to my expectation).